modules/stb/0.0.0-20241109-5c20573/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - ubuntu2404
+    - windows
+  bazel: [7.x, 8.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - '@stb//...'
```

Comment:
Since these are header-only libraries, please add `--features=parse_headers --process_headers_in_dependencies` to get at least some CI coverage.